app/core/Tracker2/index.js

```diff
@@ -18,21 +18,18 @@ export default class Tracker2 extends BaseTracker {
 
     this.store = store
 
-    // TODO consent status needs to be propagated to other trackers
     this.cookieConsentTracker = new CookieConsentTracker(this.store)
 
     this.legacyTracker = new LegacyTracker(this.store, this.cookieConsentTracker)
-    this.segmentTracker = new SegmentTracker()
+    this.segmentTracker = new SegmentTracker(this.store)
     this.googleAnalyticsTracker = new GoogleAnalyticsTracker()
     this.driftTracker = new DriftTracker(this.store)
 
     this.trackers = [
       this.legacyTracker,
-      this.googleAnalyticsTracker,
+      // this.googleAnalyticsTracker,
```

Comment:
Should this tracker be commented out and initialized above?

Reply:
Yeah no harm in initializing as the constructors just make sure to capture necessary references (like store).  Real initializing happens during the initialize calls.  I commented this out because it was technically doing nothing so I figured I'd not include it.  To that end I'll also comment out to constructor call as it's not doing anything either :)